PopApp/Components/Attendee.js

```diff
@@ -1,5 +1,5 @@
 import React from 'react'
-import {StyleSheet, View, Text} from 'react-native'
+import { StyleSheet, View, Text } from 'react-native'
```

Comment:
Seems there are import style discrepancies (spaces vs no spaces in imports)? (Discrepancy 1)